integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/PostVirksomhetPutMessageStrategy.java

```diff
@@ -9,22 +9,32 @@
 import no.difi.meldingsutveksling.ptv.CorrespondenceAgencyConfiguration;
 import no.difi.meldingsutveksling.ptv.CorrespondenceAgencyMessageFactory;
 import no.difi.meldingsutveksling.ptv.CorrespondenceRequest;
+import no.difi.meldingsutveksling.ptv.mapping.CorrespondenceAgencyValues;
+import no.difi.meldingsutveksling.serviceregistry.ServiceRegistryLookup;
+import no.difi.meldingsutveksling.serviceregistry.externalmodel.InfoRecord;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
 import static no.difi.meldingsutveksling.noarkexchange.PutMessageMarker.markerFrom;
 
 public class PostVirksomhetPutMessageStrategy implements PutMessageStrategy {
 
     private final CorrespondenceAgencyConfiguration config;
+    private final ServiceRegistryLookup serviceRegistryLookup;
 
-    public PostVirksomhetPutMessageStrategy(CorrespondenceAgencyConfiguration config) {
+    public PostVirksomhetPutMessageStrategy(CorrespondenceAgencyConfiguration config, ServiceRegistryLookup serviceRegistryLookup) {
         this.config = config;
+        this.serviceRegistryLookup = serviceRegistryLookup;
     }
 
     @Override
     public PutMessageResponseType putMessage(PutMessageRequestType requestType) {
         final PutMessageRequestWrapper putMessageWrapper = new PutMessageRequestWrapper(requestType);
+        InfoRecord senderInfo = serviceRegistryLookup.getInfoRecord(putMessageWrapper.getEnvelope().getSender().getOrgnr());
```

Comment:
Bruk heller putMessageWrapper.getSenderPartynumber og tilsvarende metode for getReceiver